src/category_theory/limits/cones.lean

```diff
@@ -513,6 +513,30 @@ functorially. -/
 def map_cocone_morphism {c c' : cocone F} (f : c ⟶ c') :
   H.map_cocone c ⟶ H.map_cocone c' := (cocones.functoriality F H).map f
 
+/-- Given a cone isomorphism `c ≅ c'`, construct a cone isomorphism on the mapped cones. -/
+def map_cone_iso {c c' : cone F} (i : c ≅ c') : H.map_cone c ≅ H.map_cone c' :=
```

Comment:
I think it's a bit silly that we have `cones.functoriality`, but then a bunch of defs and lemmas to re-do all the functor operations again - can we instead rename `cones.functoriality` to `map_cone` or something, and prevent needing extra defs like this one?

Reply:
Fair point. I'll do the refactor.